localsettings.example.py

```diff
@@ -183,8 +183,8 @@
 REPORT_CACHE = 'default'  # or e.g. 'redis'
 
 redis_cache = {
-    'BACKEND': 'redis_cache.cache.RedisCache',
-    'LOCATION': 'localhost:6379:0',
+    'BACKEND': 'django_redis.cache.RedisCache',
+    'LOCATION': 'redis://127.0.0.1:6379/0',
```

Comment:
that's correct: http://niwinz.github.io/django-redis/latest/#_configure_as_cache_backend
it's the new connection string